service/meta.proto

```diff
@@ -28,67 +31,87 @@ message Version {
 message TokenLifetime {
   // Expiration Epoch
   uint64 exp = 1;
+
   // Not valid before Epoch
   uint64 nbf = 2;
+
   // Issued at Epoch
   uint64 iat = 3;
 }
 
 // NeoFS session token.
 message SessionToken {
+  // Session token body
   message Body {
     // ID is a token identifier. valid UUIDv4 represented in bytes
     bytes id = 1;
+
     // OwnerID carries identifier of the session initiator.
-    refs.OwnerID owner_id = 2;
+    neo.fs.v2.refs.OwnerID owner_id = 2;
+
     // Verb is an enumeration of session request types
     enum Verb {
+      // Unknown verb
+      VERB_UNSPECIFIED = 0;
+
       // Refers to object.Put RPC call
-      OBJECT_PUT = 0;
+      VERB_PUT = 1;
+
       // Refers to object.Get RPC call
-      OBJECT_GET = 1;
+      VERB_GET = 2;
+
       // Refers to object.Head RPC call
-      OBJECT_HEAD = 2;
+      VERB_HEAD = 3;
+
       // Refers to object.Search RPC call
-      OBJECT_SEARCH = 3;
+      VERB_SEARCH = 4;
+
       // Refers to object.Delete RPC call
-      OBJECT_DELETE = 4;
+      VERB_DELETE = 5;
+
       // Refers to object.GetRange RPC call
-      OBJECT_RANGE = 5;
+      VERB_RANGE = 6;
+
       // Refers to object.GetRangeHash RPC call
-      OBJECT_RANGEHASH = 6;
+      VERB_RANGEHASH = 7;
```

Comment:
`OBJECT` prefix considered that this enum can be expanded for non object related tokens, therefore non object related verbs. I think it was good idea.

Reply:
I'd propose to separate contexts by type, i.e. have different context structures for each service type. Maybe we should keep verb enum as is for now and change it in the following PRs.